### ISSUE TITLE
Openssl3 support

### ISSUE THREAD

**Issue by philiphayward** [cbrandlehner/homebridge-daikin-tempsensor-nocloud#126]
I think this plugin needs the same OpenSSL3 SSL_OP_ALLOW_UNSAFE_LEGACY_RENEGOTIATION option that homebridge-daikin-local has. Tried installing and got the following.
Thanks

```
[Outside Temperature] _doSendGetRequest: ERROR: API request to https://xx.xx.xx.xx/aircon/get_sensor_info returned error Error: write EPROTO 105014A2FFFF0000:error:0A000152:SSL routines:final_renegotiate:unsafe legacy renegotiation disabled:../deps/openssl/openssl/ssl/statem/extensions.c:922:

    at WriteWrap.onWriteComplete [as oncomplete] (node:internal/stream_base_commons:94:16) {
  errno: -71,
  code: 'EPROTO',
  syscall: 'write',
  response: undefined,
  retries: 0
}
```

**Comment by cbrandlehner** [comment]
true.

**Comment by cbrandlehner** [comment]
Fixed with version 1.0.1.
